console-subscriber/src/builder.rs

```diff
@@ -101,4 +104,41 @@ impl Builder {
     pub fn build(self) -> (TasksLayer, Server) {
         TasksLayer::build(self)
     }
+
+    /// Configures this builder from a standard set of environment variables:
+    ///
+    /// * `TOKIO_CONSOLE_RETENTION`: The number of seconds to accumulate
+    ///   completed tracing data. Default: 60s.
+    /// * `TOKIO_CONSOLE_BIND`: a HOST:PORT description, such as
+    ///   localhost:1234 or similar. Default: 127.0.0.1:6669
+    /// * `TOKIO_CONSOLE_PUBLISH_INTERVAL`: The number of milliseconds to wait
+    ///   between sending updates to the console. Default: 1000ms (1s)
```

Comment:
take it or leave it: it might be nice if this was a markdown table like
```markdown
+---------------------------------+-----------------+-----------------------+
| **Environment Variable** | **Purpose** | **Default Value** |
+---------------------------------+-----------------+-----------------------+
...
```
rather than a bulleted list? but, not a big deal.

Reply:
there are also a set of public associated constants for the default values of these parameters, in case users need to set them programmatically: https://github.com/tokio-rs/console/blob/103021e5dfb7d5a25381b2edbe006e9e85a676a9/console-subscriber/src/builder.rs#L31-L35

it might be worth linking to them here, like the other `Builder` methods do?

Reply:
tables addressed in f2b6de51565b504f83b784f6de8f891fab929578

Regarding the constants, it seems like not particularly relevant to the from_default_env interface, as there's no way to programmatically use them here, but they are documented where they can be programmatically used. Does that make sense?

Reply:
yeah, i don't think it's terribly important to have them there, just a thought, not a blocker.